docs/extension/controllers.md

```diff
@@ -41,19 +47,14 @@ If you **import or recover** your Universal Profile, it will result in the addit
 Please do not **reinstall or reset** your extension if you do not have either of the following safeguards:
 
 - [2FA Recovery set up correctly](./guides/check-2fa-recovery.md)
+- [Wallet Recovery](./guides/set-wallet-recovery.md)
```

Comment:
Can we call it `3rd party wallet recovery`? Even today, people calls the UP browser extension "UP Wallet", so we want to be clear that this is using other 3rd party wallets